Source/JavaScriptCore/runtime/JSGlobalObject.cpp

```diff
@@ -1067,10 +1067,8 @@ void JSGlobalObject::init(VM& vm)
     m_regExpConstructor.set(vm, this, regExpConstructor);
     m_linkTimeConstants[static_cast<unsigned>(LinkTimeConstant::RegExp)].set(vm, this, regExpConstructor);
     m_regExpGlobalData.cachedResult().record(vm, this, nullptr, jsEmptyString(vm), MatchResult(0, 0));
-    
-    AtomicsObject* atomicsObject = nullptr;
-    if (Options::useSharedArrayBuffer())
-        atomicsObject = AtomicsObject::create(vm, this, AtomicsObject::createStructure(vm, this, m_objectPrototype.get()));
+
+    AtomicsObject* atomicsObject = AtomicsObject::create(vm, this, AtomicsObject::createStructure(vm, this, m_objectPrototype.get()));
```

Comment:
Since it is now always defined, I recommend defining it lazily like `Math` in the table (See `Math` in the table and `createMathProperty` function).

Reply:
Done. Thanks for the suggestion!